pkg/acsengine/const.go

```diff
@@ -17,6 +17,8 @@ const (
 	DefaultKubernetesMasterSubnet = "10.240.0.0/16"
 	// DefaultKubernetesClusterSubnet specifies the default subnet for pods.
 	DefaultKubernetesClusterSubnet = "10.244.0.0/16"
+	// DefaultDockerBridgeSubnet specifies the default subnet for the docker bridge network for masters and agents.
+	DefaultDockerBridgeSubnet = "172.17.0.5/16"
```

Comment:
so 172.17.0.5 makes more sense the 172.17.0.0? I just think the default currently is 172.17.0.0 so I was trying to understand why we would use a different default here than what happened before this PR. I'm checking that on provision of a new cluster what the ip is.